godot-kotlin/godot-library/src/nativeCore/kotlin/godot/core/Variant.kt

```diff
@@ -5,7 +5,7 @@ import kotlinx.cinterop.CValue
 
 class Variant (val handle: CValue<godot_variant>) {
     constructor() : this(new()) {}
-    constructor(value: Any) : this(new()) //TODO: replace
+    constructor(value: Any) : this(wrapUnsafe(value)) //TODO: replace
```

Comment:
@chippmann can we drop this in favor of `Variant.wrap(value)`? It's more symmetrical with `variant.unwrap<T>()`. At the moment this is only being used in the generated entry files.

Reply:
sure! was just a placeholder

Reply:
I'll work on that on a separate PR :)